src/Types/BigIntType.php

```diff
@@ -47,15 +44,15 @@ public function convertToPHPValue(mixed $value, AbstractPlatform $platform): int
             return $value;
         }
 
-        if ($value > PHP_INT_MIN && $value < PHP_INT_MAX) {
-            return (int) $value;
-        }
```

Comment:
Please keep this check. It will find the most common cases and should be faster than casting back and forth.

Reply:
done